jujugui/static/gui/src/app/components/profile/bundle-list/_bundle-list.scss

```diff
@@ -0,0 +1,27 @@
+.profile-bundle-list {
+  width: 766px;
+
+  &__table-header {
+    border-bottom: 1px solid $mid-grey;
+    padding-bottom: 10px;
+  }
+
+  &__row {
+    border-bottom: 1px solid $mid-grey;
+    padding-bottom: 10px;
+  }
+
+  &__row,
+  &__table-header {
+    display: grid;
+    grid-template-columns: 225px 200px 150px auto;
```

Comment:
Once a design has been made we can create media queries or percentages for these.